pkgs/tools/networking/httping/default.nix

```diff
@@ -1,16 +1,23 @@
-{ stdenv, fetchurl, gettext }:
+{ stdenv, fetchurl, gettext, openssl, ncurses, fftw }:
 
 stdenv.mkDerivation rec {
   name = "httping-${version}";
 
-  version = "2.3.4";
+  version = "2.4";
 
   src = fetchurl {
-    url = "http://www.vanheusden.com/httping/httping-2.3.4.tgz";
-    sha256 = "1hkbhdxb0phrvrddx9kcfpqlzm41xv9jvy82nfkqa7bb0v5p2qd7";
+    url = "http://www.vanheusden.com/httping/httping-2.4.tgz";
+    sha256 = "dab59f02b08bfbbc978c005bb16d2db6fe21e1fc841fde96af3d497ddfc82084";
   };
 
-  buildInputs = [ gettext ];
+  buildInputs = [ gettext openssl ncurses fftw ];
+
+  configureFlags = [
```

Comment:
Any pointer on the documentation on how to do so?

On 03/23/2015 11:18 AM, Arseniy Seroka wrote:
> In pkgs/tools/networking/httping/default.nix
> <https://github.com/NixOS/nixpkgs/pull/6923#discussion_r26925295>:
>
> >    };
> >
> > -  buildInputs = [ gettext ];
> > +  buildInputs = [ gettext openssl ncurses fftw ];
> > +
> > +  configureFlags = [
>
> Maybe make it optional?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NixOS/nixpkgs/pull/6923/files#r26925295>.
>

-- 
Best regards,
-- NP